### PR TITLE
new style ceil function

### PR DESCRIPTION
**Pull request by kmaehashi**
This PR implements new style `ceil` function.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Rebased and removed Ceil as its backward always returns zero.

**Comment by takagi** [comment]
Please fix flake8.

File: tests/chainer_tests/functions_tests/math_tests/test_ceil.py [hunk 1]
```diff
@@ -60,6 +46,7 @@ class TestCeil(UnaryFunctionsTestBase):
     def make_data(self):
         x = numpy.random.uniform(-10.0, 10.0, self.shape).astype(self.dtype)
         gy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        self.ggy = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
```
Comment (takagi) [inline tests/chainer_tests/functions_tests/math_tests/test_ceil.py]: This line can be removed.

**Comment by kmaehashi** [comment]
Thanks! Fixed.

**Pull request merged**